src/main/java/com/bathanh/exercise/basic/number/Exercise02.java

```diff
@@ -0,0 +1,15 @@
+package com.bathanh.exercise.basic.number;
+
+public class Exercise02 {
+
+    public int calculateSumOfDigits(final int a){
+        int Input = a;
+        int sum = 0, count = 0;
+        while (Input != 0){
+            count = Input % 10;
+            sum += count;
```

Comment:
Biến count thật ra không cần thiết và thậm chí nó đang được dùng sai tên gọi


```suggestion
            sum += input % 10;
```

Reply:
Resolved